### PR TITLE
Gabriel2

### PR DESCRIPTION
**Pull request by GabrielParle**

### CODE REVIEW

File: AgentDiscoveries-Frontend/package-lock.json [hunk 1]
```diff
@@ -28,6 +28,27 @@
         }
       }
     },
+    "@babel/runtime-corejs2": {
```
Comment (Wh1rr) [inline AgentDiscoveries-Frontend/package-lock.json:31]: what.

File: AgentDiscoveries-Frontend/package-lock.json [hunk 2]
```diff
@@ -12417,7 +12446,7 @@
     "warning": {
       "version": "3.0.0",
       "resolved": "https://registry.npmjs.org/warning/-/warning-3.0.0.tgz",
-      "integrity": "sha1-MuU3fLVy3kqwR1O9+IIcAe1gW3w=",
+      "integrity": "sha512-jMBt6pUrKn5I+OGgtQ4YZLdhIeJmObddh6CsibPxyQ5yPZm1XExSyzC1LCNX7BzhxWgiHmizBWJTHJIjMjTQYQ==",
```
Comment (Wh1rr) [inline AgentDiscoveries-Frontend/package-lock.json:12449]: why.

File: AgentDiscoveries-Frontend/app/src/components/home.jsx [hunk 3]
```diff
@@ -1,8 +1,12 @@
 import * as React from 'react';
 
-import { reactComponent as Bond } from '../../static/james-bond-007.svg';
+
+//import { reactComponent as Bond } from '../../static/james-bond-007.svg';
```
Comment (Wh1rr) [inline AgentDiscoveries-Frontend/app/src/components/home.jsx:4]: rip james

**Review by Wh1rr — COMMENTED** [review]
dont really know frontend but seems all good

File: AgentDiscoveries-Backend/src/main/resources/config.properties [hunk 4]
```diff
@@ -1,8 +1,8 @@
 
 # Database properties
-database.url = jdbc:mysql://localhost:3306/agentdiscoveries?useSSL=false
-database.username = agentdiscoveries
-database.password = agentdiscoveries
+database.url = jdbc:mysql://localhost:3306/agentdiscoveries
+database.username = root
+database.password = root
```
Comment (RogVan) [inline AgentDiscoveries-Backend/src/main/resources/config.properties:5]: Do we want to commit these?

File: AgentDiscoveries-Frontend/app/src/components/home.jsx [hunk 5]
```diff
@@ -1,13 +1,54 @@
 import * as React from 'react';
 
-import { Redirect } from 'react-router-dom';
+
+//import { reactComponent as Bond } from '../../static/james-bond-007.svg';
+
+//import { Redirect } from 'react-router-dom';
```
Comment (RogVan) [inline AgentDiscoveries-Frontend/app/src/components/home.jsx:6]: Remember to delete comments if you aren't using them

File: AgentDiscoveries-Frontend/app/src/components/home.jsx [hunk 6]
```diff
@@ -1,13 +1,54 @@
...
+import {Link} from 'react-router-dom';
+
+
 
 export default class Home extends React.Component {
+
+
+
+
```
Comment (RogVan) [inline AgentDiscoveries-Frontend/app/src/components/home.jsx:15]: Probably don't need all the whitespace lines in this file, one separating line at most is the usual policy

File: AgentDiscoveries-Frontend/app/src/components/home.jsx [hunk 7]
```diff
@@ -1,13 +1,54 @@
...
+        // Get the canvas node and the drawing context
+
+
     }
+
 }
+
+
+
```
Comment (RogVan) [inline AgentDiscoveries-Frontend/app/src/components/home.jsx:54]: Same deal with the spare lines here

File: AgentDiscoveries-Frontend/app/src/components/page.jsx [hunk 8]
```diff
@@ -1,13 +1,16 @@
...
+import background from '../../static/cool-background.png';
 
 export default class Page extends React.Component {
     render() {
         return (
             <React.Fragment>
-                <NavigationBar />
-                <div className='container'>
+                
+
```
Comment (RogVan) [inline AgentDiscoveries-Frontend/app/src/components/page.jsx:11]: Lines again here

File: AgentDiscoveries-Frontend/app/src/index.jsx [hunk 9]
```diff
@@ -2,10 +2,16 @@ import * as React from 'react';
...
+//import '~bootstrap/scss/bootstrap';
+
+
 
 import '../sass/styles.scss'; // Import so that webpack loads all of the sass
 
 ReactDOM.render(
+
+    
+
```
Comment (RogVan) [inline AgentDiscoveries-Frontend/app/src/index.jsx:14]: And here

**Review by RogVan — CHANGES_REQUESTED** [review]
Nice, looks really good! Just a couple of small things to tidy up the code.

Make sure to delete any of the image files you aren't planning on using, as they can quickly make the repo really big

**Pull request closed without merging**